tailcfg/tailcfg.go

```diff
@@ -34,7 +34,7 @@ import (
 // Previously (prior to 2022-03-06), it was known as the "MapRequest
 // version" or "mapVer" or "map cap" and that name and usage persists
 // in places.
-type CapabilityVersion int
+type CapabilityVersion uint16
```

Comment:
I don't think this will be worth the pain.

Just explode elsewhere if this ever goes over uint16 in size.  (or write some test that fails if it goes over 64k)

Reply:
Switched tailcfg type back to int. I kept uint16 in controlbase and controlhttp, you okay with that?

Added a test that will fail when tailcfg version exceeds 60k, as an early warning to revise our wire protocol. Also added a hard panic() in controlclient if we ever try to start a session with a >uint16 version. Hard panic because the test should give us very early warning that the panic is coming, so something's very wrong if that codepath trips.